step-generation/src/commandCreators/atomic/dispenseInPlace.ts

```diff
@@ -22,20 +33,25 @@ export const dispenseInPlace: CommandCreator<DispenseInPlaceParams> = (
       },
     },
   ]
-
   const pipettePythonName =
     invariantContext.pipetteEntities[pipetteId].pythonName
   const pythonArgs = [
     `volume=${volume}`,
     // rate= is a ratio in the PAPI, and we have no good way to figure out what
-    // flowrate the PAPI has set the pipette to, so we just have to emit a division:
+    // flowrate the PAPI has set the pipette to, so we just have to do a division:
     `rate=${flowRate} / ${pipettePythonName}.flow_rate.dispense`,
-    ...(pushOut != null ? [`push_out=${pushOut}`] : []),
+    ...(pushOut != null ? [`pushOut=${pushOut}`] : []),
```

Comment:
i think it was right before? 
```suggestion
    ...(pushOut != null ? [`push_out=${pushOut}`] : []),
```

Reply:
Thank you for the catch— yep, should be snake case